css-ui-3/caret-color-016.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color visited link computed value</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<link rel="help" href="https://www.w3.org/TR/css3-color/#color0">
+<meta name="flags" content="dom interact">
```

Comment:
Note that using `href="caret-color-016.html"` makes that you don't need to visit it in some browsers (so it will just run automatically).

I don't know either what's the best approach, but we'd like to have a test to verify this on Blink. So we can do it as part of the W3C suite or just on Blink.